py/desispec/io/meta.py

```diff
@@ -130,7 +130,7 @@ def get_raw_files(filetype, night, expid, rawdata_dir=None):
             guaranteed to match the regular expression [brz][0-9].
     """
     glob_pattern = findfile(filetype, night, expid, camera='*', rawdata_dir=rawdata_dir)
-    literals = map(re.escape,glob_pattern.split('*'))
+    literals = list(map(re.escape,glob_pattern.split('*')))
```

Comment:
Good point.  I implemented the 2to3 recommendation without checking, but you are right that the list() wrapper isn't necessary in this case.  `multiprocessing.Pool.map` and vanilla `map` accepts an iterable for the function arguments and doesn't require it to be a list.  I'll wait for other comments and update them in bulk.